tests/integration/targets/aci_fabric_interface_policy_group/tasks/main.yml

```diff
@@ -0,0 +1,327 @@
+    state: present
+  check_mode: true
+  register: cm_spine_policy_group_present
+
+- name: Execute tasks only for ACI v6.0(2h)+
+  when: version.current.0.topSystem.attributes.version is version('6.0(2h)', '>=')
+  block:
+    - name: Add a spine fabric interface policy group with transceiver_policy_tdn
+      aci_fabric_interface_policy_group: &tp_spine_policy_group_present
+        <<: *aci_info
+        name: spine_policy_group
+        type: spine
+        descr: spine_policy_group created
+        transceiver_policy_tdn: "uni/fabric/zrfab-default"
+        state: present
+      register: tp_spine_policy_group_present
+
+    - name: Remove the transceiver_policy from the spine fabric interface policy group
+      aci_fabric_interface_policy_group:
+        <<: *aci_info
+        name: spine_policy_group
+        type: spine
+        descr: spine_policy_group created
+        transceiver_policy_tdn: ""
+        state: present
+      register: tp_spine_policy_group_absent
+
+    - name: Assertions check for add fabric interface policy groups
+      assert:
+        that:
+          - tp_spine_policy_group_present is changed
+          - tp_spine_policy_group_absent is changed
+          - tp_spine_policy_group_present.current[0].fabricSpPortPGrp.children  | selectattr('fabricRsOpticsFabIfPol.attributes.tDn', 'defined') | selectattr('fabricRsOpticsFabIfPol.attributes.tDn', 'equalto', "uni/fabric/zrfab-default")
+          - tp_spine_policy_group_absent.current[0].fabricSpPortPGrp.children  | selectattr('fabricRsOpticsFabIfPol.attributes.tDn', 'defined') | selectattr('fabricRsOpticsFabIfPol.attributes.tDn', 'equalto', "")
+
+- name: Add a spine fabric interface policy group with normal mode
+  aci_fabric_interface_policy_group:
+    <<: *cm_spine_policy_group_present
+  register: nm_spine_policy_group_present
+
+- name: Add a spine fabric interface policy group with normal mode again
+  aci_fabric_interface_policy_group:
+    <<: *cm_spine_policy_group_present
+  register: nm_spine_policy_group_present_again
+
+- name: Add a spine fabric interface policy group with normal mode again
+  aci_fabric_interface_policy_group:
+    <<: *cm_spine_policy_group_present
+  register: nm_spine_policy_group_present_again
+
+- name: Update a spine fabric interface policy group with check mode
+  aci_fabric_interface_policy_group: &cm_spine_policy_group_update
+    <<: *cm_spine_policy_group_present
+    descr: spine_policy_group updated
+  check_mode: true
+  register: cm_spine_policy_group_update
+
+- name: Update a spine fabric interface policy group with normal mode
+  aci_fabric_interface_policy_group:
+    <<: *cm_spine_policy_group_update
+  register: nm_spine_policy_group_update
+
+- name: Update a spine fabric interface policy group with normal mode again
+  aci_fabric_interface_policy_group:
+    <<: *cm_spine_policy_group_update
+  register: nm_spine_policy_group_update_again
+
+- name: Assertions check for add fabric interface policy groups
```

Comment:
Can you test the following if not done already?
 dwdm_policy: default
    link_level_policy: default
    link_flap_policy: default
    l3_interface_policy: default
    macsec_policy: default
    monitoring_policy: default